### ISSUE TITLE
Create helper funcs for `enable_rhrepo` and `disable_rhrepo`

### ISSUE THREAD

**Issue by abalakh** [SatelliteQE/robottelo#2638]
Per #2632 discussion.
To enable/disable RH repository, we usually do the same repeatable steps, like
* Find product by name & org.id
* Find repository set by name & product id
* Enable/disable repo by reposet id & product id

All above should be covered by helper function. We already have `enable_rhrepo_and_fetchid()` in `robottelo/api/utils.py`, but
* It only enables repo, we still need `disable` func
* I'm not sure whether `robottelo/api/utils.py` is the best place for it, i'd think about `robottelo/common/helpers.py`
* It's not factory-like: works with reposet & product names. But what if we already know reposet & product ids? We don't need to search for entities in that case, and org.id parameter is redundant.
* It returns enabled repo id. That's a good thing, but it requires passing repo_name as required parameter, and we don't always need that id and not always know the repo name, so this functionality should be optional.

Summarizing, `enable_rhrepo_and_fetchid()` is not completely covering all the needs, so better helper funcs `enable_rhrepo` and `disable_rhrepo` should be created instead.

**Comment by elyezer** [comment]
:+1:

**Comment by kbidarkar** [comment]
Please note that for RedHat Repositories you cannot get to know or there is no way for you to get to know the `RedpoSet ID` or `Product ID`. What you will ever be aware of is only and only "RepoSet Names` and `Product Names`. This is the only way to fetch the REPO IDs.

As this function clearly states this is just for `Enabling RedHat RepoSets, Products` and fetching it's `Repo ID`.

Please `DO NOT EXPECT` factroy like stuff for this function as ID can never be known for RedHat Repos beforehand, it can only be known beforehand for `CUSTOM Repos` Only.

Yes we can improve this function for the below 2 things.
a) disable func
b) debate where to place it `helpers.py` or retain it at `utils.py`.

**Comment by kbidarkar** [comment]
Also repo id can only be fetched , if the repo is enabled and you need to know repo-name.
To enable repo we require, reposet name and product name and hence the function.

Please note all are tied up together, the only way is to enable things is via `NAMES`.

**Comment by omaciel** [comment]
@abalakh lovely suggestion and you have my :100: to make it happen. However, as @kbidarkar pointed out, one cannot really know what the ID will be for Red Hat repositories, so you'll still have to do some heavy lifting there. But I'd love to see a helper that is easy to use and intuitive, so I am looking forward to seeing what you put together :)

**Comment by kbidarkar** [comment]
:+1:  for improving the code,  I was thinking only from functionality aspect.
It would really be good to see the function/code evolve/improve. Let's keep raising such issues to help improve/make code intuitive.

**Comment by abalakh** [comment]
@kbidarkar you're right, to know the id you have to search by name first. What i was pointing at is we have few tests which are enabling/disabling some reposet, for example. To enable reposet you have to search by name, find the ids and enable the reposet. Then, to disable it, you don't really need to search by its name, as you already have the ids and could skip the search. That was the area which can be impoved.
Although I think making `enable_rhrepo_and_fetchid()` a factory function may be not the best idea - for different input data we have different required arguments. E.g. if we pass product/reposet ids the `org_id` is redundant, but for product/reposet names it is required. So accepting both names and ids will make the function less intuitive.

**Comment by abalakh** [comment]
Regarding where to put `enable_rhrepo_and_fetchid()` - I think `robottelo/api/utils.py` is fine for now.
At the moment when issue was risen `enable_rhrepo_and_fetchid()` was the only function in `robottelo/api/utils.py`, that's why i was thinking about moving the function and removing the redundant module.
Right now there're few more helper functions in the module, all of them are working with API (unlike helpers from  `robottelo/common/helpers.py`) so separate module is fine for them.
